src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -179,6 +179,16 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     # NO_EMBEDDED_PT argument (for example, if you need to chain into the binary, it
     # can't contain a partition table).
     #
+    # This will create additional UF2 files for the CYW43 firmware and a TBYB version of the
+    # CYW43 firmware. You will need to flash one of these to each device once, after loading the
+    # partition table. For example using picotool:
+    #
+    # `picotool load TARGET.uf2`;
+    # `picotool reboot -u`;
+    # `picotool load TARGET_wifi_firmware.uf2 -x`;
```

Comment:
Having just looked at the [picotool README](https://github.com/raspberrypi/picotool/tree/develop?tab=readme-ov-file#load), perhaps it's worth suggesting the `-u` flag here too, so that they don't need to re-write the CYW43 firmware if it has already been written? :thinking:

Reply:
Also, I dunno if it's just a "me thing" but I always tend to put flags first and filenames last, so personally I'd write this as:
```
picotool load -x TARGET_wifi_firmware.uf2
```